### DISCUSSION TITLE
Specific atom force error

### DISCUSSION THREAD

**Discussion by ald3hyde** [deepmodeling/deepmd-kit#2704] [General]
Dear all
I try a multi-component molten slag system now. Since then(#2663), I have tried to think and devise sampling methods, but there has been no improvement at all. I have improved the cutoff to 15.0 and so on but to no avail. the force error is spooky, but the agreement on the RDF plot is excellent. Does it seem like there is a problem with the ab initio calculations? Any advice would be appreciated.
input.json.
・DeePMD-kit v2.2.2
・Tensorflow 2.90
・GPU:2070Ti
・CUDA:11.2

  "model": {
    "type_map": [
      "Si",
      "Al",
      "B",
      "Na",
      "O"
    ],
       "use_srtab": "zbl.dat",
      "smin_alpha": 0.1,
      "sw_rmin": 0.9,
      "sw_rmax": 1.0,
      "type_embedding": {"neuron": [2, 4, 8],
			 "activation_function": "tanh",
			 "trainable": true}, 
    "descriptor": {
	"type": "se_atten",
	"sel": 300,
	"rcut": 6.0,
	"rcut_smth": 0.5,
	"neuron": [
            25,
            50,
            100
	],
      "axis_neuron": 8,
      "resnet_dt": false,
      "seed": 0,
      "attn": 128,
      "attn_layer": 0,
      "attn_dotr": true,
      "attn_mask": false
    },
    "fitting_net": {
      "type": "ener",
      "neuron": [
        240,
        240,
        240
      ],
      "resnet_dt": false,
      "seed": 1
    }
  },
  "learning_rate": {
    "type": "exp",
    "start_lr": 0.001,
    "stop_lr": 3.51e-8,
    "decay_steps": 5000
  },
  "loss": {
      "type": "ener",
      "start_pref_e": 0.02,
      "limit_pref_e": 1,
      "start_pref_f": 20000,
      "limit_pref_f": 1,
      "start_pref_v": 0,
      "relative_f": 1,
      "limit_pref_v": 0
  },
- SBAN_1(3000K）
- SBAN_2(200０K）
- SBN（3000K）
- SBAN_2(300K)
these plot data were generated by the dp test -m model.pb -s path/to/system -d name -n 30
![image](https://github.com/deepmodeling/deepmd-kit/assets/123191059/eefb396b-89bc-4b49-af66-98eb53c322dc)
![image](https://github.com/deepmodeling/deepmd-kit/assets/123191059/b5e43cf6-a667-4a47-b5ad-26a296deea2d)
![Pasted Graphic 1](https://github.com/deepmodeling/deepmd-kit/assets/123191059/3f3af905-1186-4012-9ded-af4d3a1be09f)
・Na only plot(metrics are the same values as the former)
![image](https://github.com/deepmodeling/deepmd-kit/assets/123191059/b91595d3-e6ae-4f9f-9d22-adf5689ec771)
・If using OC_2M pretraining model
trangely enough, the RMSE of Na improved, but the RMSEs of Si and B error as Na of the non-pretraining model.
![image](https://github.com/deepmodeling/deepmd-kit/assets/123191059/108315b3-f7db-47d2-aaf3-b5141c2e872a)

**Comment by njzjz** [comment]
What's the cutoff energy in your ab initio calculation? Could you increase the accuracy and see if the results differ?

**Reply by ald3hyde** [reply]
Thank you, Dr.Zeng.
The energy cutoffs are between 75 Ry and 85 Ry.
We will calculate AIMD at an high accuracy asap.

**Reply by ald3hyde** [reply]
The results improved when higher-level DFT calculations (around 1200 eV) were used as training data. Thanks!

**Reply by ald3hyde** [reply]
120 Ry and 1e-9 Ry(SCF)
